### ISSUE TITLE
[For Jan 2026] Content review of iframe titles

### ISSUE THREAD

**Issue by helennickols** [ministryofjustice/moj-frontend#2025]
## What

The iframe titles are not descriptive or unique. This doesn't count as a WCAG fail but should be improved. For example, on the alert we have titles and they are Alert (example). Another example: iFrame used to add Google Maps to the page. The iFrame has no title whatsoever and therefore has no accessible name.

This is a usability issue that came out of the review of the a11y statement in November 2025. We have committed in the statement to fix this by Jan 2026. 

## Why

Usability. 

## Who needs to be involved

- Interaction Designer
- Content Designer
- Developer
- Accessibility Specialist

## Done

This is done when the titles have been changed and are unique. This is done when the reference to it needs to be removed from the a11y statement under the following headings:

### Non-compliance with the accessibility regulations

**Comment by helennickols** [comment]
'Add another' iframe work taken out into the improvement work for that component.

**Comment by chrispymm** [comment]
To add to this slightly, the title that is used for the iframes (and other places as @timminor noted) simply pulls from the example nunjucks file's frontmatter.  A lot of them are duplicated as one example is often created by copying from one to another, and the title doesn't get modified.  So the easiest and simplest fix for this ticket is to just make sure each example file has a unique title. 

Maybe you both already knew all this, but just thought i'd add it in in case it was helpful.

@helennickols 👆those alert examples sound good 🙂

**Comment by helennickols** [comment]
Howdy, I've added proposed iframe titles to the [doc](https://justiceuk-my.sharepoint.com/:w:/r/personal/helen_nickols_justice_gov_uk/Documents/to%20delete%20in%20feb%202026%20-%20iframe%20titles.docx?d=wdea575d95e9f4d1ca4e460e776760c01&csf=1&web=1&e=SfhKno) for review. Please add comments in the doc. I've removed the labels for the iframes to reduce confusion. They should be in the same order that they appear in the documentation. 

@chrispymm There's a question in there for you about the button menu.

**Comment by chrispymm** [comment]
@helennickols looks good 👍  I've made a couple of small comments

**Comment by helennickols** [comment]
Hi @RoseBrandle72 would you mind reviewing the iframe titles please? Let me know if anything is unclear! I'm also sending to Eu for a review.
